pkg/forward/forwarder.go

```diff
@@ -46,6 +50,18 @@ func (f *Forwarder) Forward(ctx context.Context, spec Spec) error {
 		return err
 	}
 
+	ch, _ := signal.NotifyContext(ctx, os.Interrupt, syscall.SIGTERM)
```

Comment:
This belongs in `package cmd` since it deals with command line inputs. From there, this method should instead attempt to delete the pod if/when the context is canceled. Alternatively, a forwarder could behave statefully and expose a separate `Stop` method.